src/app/modules/OpenshiftStreams/components/StreamsTableConnected/StreamsTableConnected.tsx

```diff
@@ -107,6 +110,32 @@ export const StreamsTableConnected: VoidFunctionComponent<
 
   const [shouldRefresh, setShouldRefresh] = useState(false);
 
+  //kafka size state
+  const [kafkaSize, setKafkaSize] =
+    useState<CreateKafkaInstanceWithSizesTypes.GetSizesData>();
```

Comment:
This type isn't the right one, it should come from the hook, not from the UI components.